02week/blog/index.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" href="css/style.css">
+  <title>DIY Shoegazer | Guitar peal reviews</title>
+</head>
+
+
+<body>
+  <div class="container">
+    <div class="header">
+      <h1>DIY SHOEGAZER</h1>
+    </div>
+    <div class="left-column">
+      <img src="">
+      <br>
+      <ul>
+        <li>
+          <a href="/archive">Archive</a>
+        </li>
+        <li>
+          <a href="/subscribe">Subscribe</a>
+        </li>
+        <br>
+        <li>
+          <a href="/submitpost">Submit a post</a>
+        </li>
+      </ul>
+      <div class="description">
+        self proclaimmed shoegazer, I, matt guess, review my favorite (and pretty much whatever I get my hands on) guitar pedals, rack f/x, budget home
+        studio recording equipment and techniques, floss digital vs. analog, spend way too much time discussing the art of compression and mastering, and
+        share my lust for vintage jazzmasters and excessive reverb. I also play in raleigh shoegaze band white cascade and solo as jazz tapes
+      </div>
+      <div>
+        <form action="/action_page.php" method="get">
```

Comment:
curious if you copied and pasted this from somewhere? or if you have a server running somewhere

Reply:
I definitely peaked at the code for the tumblr blog I was recreating and copied this particular code for the form.